### ISSUE TITLE
Adicionar vários exemplos de código

### ISSUE THREAD

**Issue by armindomatheus** [levxyca/diciotech#47]
Na na expressões ternárias temos a presença de um código como exemplo, porém em certos casos acho que podemos mostrar um exemplo em várias linguagens diferentes, já que dependo do exemplo a sintaxe pode mudar de linguagem pra linguagem

**Comment by thiagofqs** [comment]
Creio que é uma boa sim, mas não sei qual seria a melhor forma de implementar isso. Temos que discutir aqui qual seria o ideal e quais linguagens devem estar presentes, pois não podem ser muitas e sim talvez umas quatro.

Posso fazer um PR de uma primeira versão dessa feature! 🫡 Claro, de acordo com o que for discutido nessa issue e se a @levxyca, junto com a comunidade, concordar na implementação dessa sugestão.

**Comment by levxyca** [comment]
Eu acho válido abrir a discussão para a comunidade, posso cirar um tópico novo nos `discussions` do repositório, meu único ponto quanto a isso é fugir muito da ideia de explicar os termos em si, o código de exemplo é mais um apoio para a pessoa conseguir visualizar, mas a ideia realmente não é ensinar implementar aqui, faz sentido?

@armindomatheus @thiagofqs

**Comment by thiagofqs** [comment]
> Eu acho válido abrir a discussão para a comunidade, posso cirar um tópico novo nos `discussions` do repositório, meu único ponto quanto a isso é fugir muito da ideia de explicar os termos em si, o código de exemplo é mais um apoio para a pessoa conseguir visualizar, mas a ideia realmente não é ensinar implementar aqui, faz sentido?
> 
> @armindomatheus @thiagofqs

Concordo, @levxyca!

Bem, já deixando minha contribuição nessa discussão, penso da seguinte forma:
Mesmo o objetivo do Diciotech sendo explicar os termos em si, acho válido uma exemplificação com uma linguagem ou outra, mas muito poucas. Por exemplo, se vier algum conceito que é necessário ser exemplificado através de um código, será que é possível fazê-lo sempre no padrão como é no termo do operador ternário `Expressão ? Verdadeiro : Falso`? Pode ser que sim, mas também pode não ficar muito claro para alguma pessoa usuária que está em busca do conceito do termo.

**Comment by levxyca** [comment]
Uma sugestão, pensando em implementação, seria alinhar isso com a nova UI que está sendo desenvolvida na issue #12

**Comment by thiagofqs** [comment]
> Uma sugestão, pensando em implementação, seria alinhar isso com a nova UI que está sendo desenvolvida na issue #12

Siimm! Se realmente formos implementar, antes temos que procurar padronizar um pouco as coisas, como incluir algumas regras e requisitos para incluir novos termos no arquivo JSON.

**Comment by levxyca** [comment]
> > Uma sugestão, pensando em implementação, seria alinhar isso com a nova UI que está sendo desenvolvida na issue #12
> 
> Siimm! Se realmente formos implementar, antes temos que procurar padronizar um pouco as coisas, como incluir algumas regras e requisitos para incluir novos termos no arquivo JSON.

Exatamente, vou converter essa issue em uma discussion para nos organizarmos melhor!
